src/streams/rtx_cache.rs

```diff
@@ -1,23 +1,19 @@
-use std::collections::BTreeMap;
 use std::time::Duration;
 use std::time::Instant;
 
 use crate::io::DATAGRAM_MAX_PACKET_SIZE;
 use crate::rtp_::SeqNo;
 
+use super::rtx_cache_buf::EvictingBuffer;
 use super::RtpPacket;
 
 const RTX_CACHE_SIZE_QUANTIZER: usize = 25;
 const RTX_CACHE_QUANTIZE_SLOTS: usize = DATAGRAM_MAX_PACKET_SIZE / RTX_CACHE_SIZE_QUANTIZER;
 
 #[derive(Debug)]
 pub(crate) struct RtxCache {
-    // Config
-    max_packet_count: usize,
-    max_packet_age: Duration,
-
     // Data, new additions here probably need to be cleared in [`clear`].
-    packet_by_seq_no: BTreeMap<SeqNo, RtpPacket>,
```

Comment:
Think this is a pretty good candidate for feature flag, or option to pick one packet backend over the other. There might be tradeoffs that make people want to pay more cogs for smaller memory footprint which might be hard to replicate with the buf.

Reply:
maybe even a trait and have 2 implementations with a feature flag around

Reply:
I don't know why people would want the BTreeMap?